docs/plugins/rewrite.rst

```diff
@@ -26,6 +26,18 @@ might use::
     rewrite:
         artist .*jimi hendrix.*: Jimi Hendrix
 
+As of v1.3.14 the replacement pattern may also consist of a regular expression
```

Comment:
These docs look great; thank you!

There's no need to include the version number here, though—since the docs are versioned as a whole.